app/controllers/admin/poll/questions/answers_controller.rb

```diff
@@ -9,6 +9,7 @@ def new
 
   def create
     @answer = ::Poll::Question::Answer.new(answer_params)
+    @answer.set_order
```

Comment:
I think since there might be other ways to create an answer (think through rake task, console, etc...) and a value at the `given_order` attribute is always needed. This `set_order` method shouldn't be called here, but on a `before_create` on the Poll Question Answer model